list_etexts.php

```diff
@@ -8,28 +8,26 @@
 
 $x        = get_enumerated_param($_GET, 'x', 'g', array('g', 's', 'b'));
 $sort     = get_integer_param($_GET, 'sort',      0, 0, 5);
-$per_page = get_integer_param($_GET, 'per_page', 20, 1, NULL);
+$per_page = get_integer_param($_GET, 'per_page', 20, 1, 100);
 $offset   = get_integer_param($_GET, 'offset',    0, 0, NULL);
 
-$boilerplate = _("These e-texts are the product of hundreds of hours of labor donated by all of our volunteers. The list is sorted with the most recently submitted e-texts at the top. You can sort them based upon your own preferences by clicking below. Enjoy!!");
-
 if($x == "g") {
     $type = "gold";
     $title = _("Completed Gold E-Texts");
     $state = SQL_CONDITION_GOLD;
-    $info = _("Below is the list of Gold e-texts that have been produced on this site. Gold e-texts are books that have passed through all phases of proofreading, formatting, and post-processing. They have been submitted to Project Gutenberg and are now available for your enjoyment and download.");
+    $info = _("Below is the list of Gold e-texts that have been produced on this site. Gold e-texts are books that have passed through all phases of proofreading, formatting, and post-processing. They have been submitted to Project Gutenberg and are now available for your enjoyment and download.") . "<p></p>" . _("These e-texts are the product of hundreds of hours of labor donated by all of our volunteers. The list is sorted with the most recently submitted e-texts at the top. You can sort them based upon your own preferences by clicking below. Enjoy!!");
```

Comment:
It's not at all intuitive why we place `<p></p>` tags between the sentences (actually, it should be `</p><p>` but I digress).

It would be better if we made `$info` into an array that we surround with paragraph tags when we output them. This would also allow us to put the two sentences on separate rows for shorter lines.

e.g.
```php
$info = [
    _("Para 1"),
    _("Para 2"),
];
```

Then when we output them using this convenience function in `misc.inc`:
```php
echo surround_and_join($info, "<p>", "</p>", "");
```

Reply:
Thanks, I'll do that. And ... that `<p></p>` was an error on my part. I thought I'd carefully done `</p><p>` for all instances.

Reply:
And done.